.github/workflows/composerAudit.yaml

```diff
@@ -0,0 +1,32 @@
+on: [push]
+name: Composer Audit Check
+jobs:
+  composer-audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Read the current used PHP Version
+        id: get-php-version
+        run: echo "::set-output name=version $(head -n1 .php-version)"
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ steps.get-php-version.outputs.version }}
+          tools: pecl, composer:2
+          extensions: gd, mbstring, intl, amqp
+          coverage: xdebug
```

Comment:
xdebug wird in dem Context nicht benötigt.

Reply:
:D Ich muss mal meine Vorlage ändern.